profiles/manatime/manatime.install

```diff
@@ -10,17 +10,30 @@
 function manatime_install() {
   // Create a default role for site administrators,
   // with all available permissions assigned.
+  $super_admin_role = new stdClass();
```

Comment:
It's working now; however, by readability, I'd prefer this order:
- $super_admin_role = new stdClass();
- $super_admin_role->name = 'super admin';
- $super_admin_role->weight = 2;
- user_role_save($super_admin_role);

Then the same for every other role.